### DISCUSSION TITLE
Using Google Assistant SDK

### DISCUSSION THREAD

**Discussion by FDNYCoop** [cgiesche/streamdeck-homeassistant#237] [Help and Support]
I'm trying to integrate an ability to turn my AC on by using my stream deck, I'm not familiar with what I'm doing within the plugin yet watching tutorials online and guessing I believe I have it set correctly, using the send call via the home assistant website it works correctly I'm just struggling to understand how the JSON part works or if I'm even doing in correctly, any help would be greatly appreciated, thanks!
![image](https://github.com/cgiesche/streamdeck-homeassistant/assets/52199987/158fb656-a9de-439f-83f4-4a406d4506f3)

**Comment by cgiesche** [comment]
Hi CoopSkieZ,

the data you can enter in the field is not standardized but does depend on the integration you are using, in your case "google assistant". As the available options rely on the integration, i can not tell, which options there are and what are the available values :(

https://www.home-assistant.io/docs/scripts/service-calls/#passing-data-to-the-service-call
